tests/appsec/test_blocking_addresses.py

```diff
@@ -132,3 +132,185 @@ def test_response_header(self):
     @missing_feature(reason="No endpoint defined yet")
     def test_response_cookies(self):
         assert False
+
+
+@rfc("https://datadoghq.atlassian.net/wiki/spaces/APS/pages/2667021177/Suspicious+requests+blocking")
+@coverage.not_implemented
+@released(cpp="?", dotnet="?", php_appsec="?", python="?", nodejs="?", golang="?", ruby="?")
+class Test_Blocking_request_method:
```

Comment:
Just python style (and same for others): 

```suggestion
class Test_BlockingRequestMethod:
```